Gopkg.toml

```diff
@@ -44,10 +44,6 @@ ignored = ["github.com/Sirupsen/logrus"]
   revision = "19035310d4ba1b58056aae427ea669d1db5fc618"
   source = "github.com/machine-drivers/machine"
 
-[[constraint]]
```

Comment:
This should be removed because https://github.com/minishift/minishift/blob/master/Gopkg.toml#L183-L186 already exist and our code use that instead of this one.

Reply:
make this a specific commit please. can be numbered as part of the proxy PR.

Reply:
@praveenkumar Are you considering it? Make sense IMO